themes/bootstrap3/templates/ajax/itemLocationList.phtml

```diff
@@ -0,0 +1,12 @@
+<?php $availabilityStatus = $this->availabilityStatus(); ?>
+<?php foreach ($this->locationList as $location): ?>
+  <div class="groupLocation">
+    <span class="<?= $availabilityStatus->getClass($location['availability'])?>">
+      <?= $this->icon($availabilityStatus->getIcon($location['availability']))?>
+      <?= $this->escapeHtml($location['location'])?>
+    </span>
+  </div>
+  <div class="groupCallnumber">
+    <?= $location['callnumber']?>
+  </div>
+<?php endforeach; ?>
```

Comment:
It's probably worth adding an end-of-file line break here for style consistency.